### ISSUE TITLE
Unselect button does nothing

### ISSUE THREAD

**Issue by sopoforic** [burtonator/polar-bookshelf#1475]
2.0 beta 20 on Chrome 84.

At the top of the document repo is an empty checkbox. When documents are selected, it changes to a minus icon, and I presume it is meant to deselect all documents. However, clicking it does nothing.

**Comment by burtonator** [comment]
It's meant to be a none -> some -> all selector...

So none it is empty, some it's partially empty, and clicking it again toggles them all.

**Comment by sopoforic** [comment]
Right now (beta 25), it works to select all if none are selected, but otherwise it does nothing.

**Comment by matthewjh** [comment]
You are right. Once it is in the "all selected" state, clicking it again does nothing (it should deselect everything).
